lib/active_model/serializer.rb

```diff
@@ -14,6 +14,9 @@ class Serializer
     include Configuration
     include Associations
     require 'active_model/serializer/adapter'
+    Attribute = Struct.new(:name, :reader) do
+      delegate :call, to: :reader
+    end
```

Comment:
See https://github.com/rails-api/active_model_serializers/pull/1356#issuecomment-161193512 

I could go either way on this.